### PR TITLE
Tab bar view

### PR DESCRIPTION
**Pull request by qadriyyah119**

### CODE REVIEW

File: CoWorkMobileApp/Controllers/CollectionsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//
+//  CollectionsViewController.swift
+//  CoWorkMobileApp
+//
+//  Created by Qadriyyah Thomas on 8/10/22.
+//
+
+import UIKit
+
+class CollectionsViewController: UIViewController {
```
Comment (Marquis103) [inline CoWorkMobileApp/Controllers/CollectionsViewController.swift:10]: Why did you need to do this?

File: CoWorkMobileApp/Controllers/SearchViewController.swift [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        ]
+        gradient.locations = [0, 0.25, 1]
+        return gradient
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        gradient.frame = view.bounds
+        view.layer.addSublayer(gradient)
+//        view.backgroundColor = ThemeColors.grayColor
```
Comment (Marquis103) [inline CoWorkMobileApp/Controllers/SearchViewController.swift:28]: Still need this?

**Review by Marquis103 — APPROVED** [review]

File: CoWorkMobileApp/Controllers/CollectionsViewController.swift (hunk 1 above)
Comment (qadriyyah119) [reply CoWorkMobileApp/Controllers/CollectionsViewController.swift:10]: This is the view controller for the user's saved workspaces. Maybe I should rename it.
